src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,47 @@
+package calculator;
+
+import utils.StringUtil;
+
+import java.util.Arrays;
+
+public class Calculator {
+    public final static String INPUT_EXCEPTION_MSG="입력된 값에 오류가 있습니다.";
+
+    private String getValidExpression(String in) throws IllegalArgumentException{
+        if(in == null || in.isEmpty() || in.replace(StringUtil.WHITESPACE,"").length() == 0){
+            throw new IllegalArgumentException(INPUT_EXCEPTION_MSG);
+        }
+        return in;
+    }
+
+    public long getResult(String expression){
+        String validExpression = getValidExpression(expression);
+
+        long[] operands = getOperands(validExpression);
+        Operator[] operators = getOperators(validExpression);
+
+        long acc = operands[0];
+
+        for(int i = 1; i < operands.length; i++){
+            acc = operators[i-1].action(acc, operands[i]);
+        }
+
+        return acc;
+    }
+
+    private long[] getOperands(String expression){
+        return Arrays.stream(StringUtil.regexSplit(expression, StringUtil.REGEX_OPERATOR))
+                .filter(x->!x.isEmpty())
```

Comment:
```suggestion
                .filter(x- > !x.isEmpty())
```
다음과 같은 space 컨벤션은 IDE 의 **reformat** 기능을 활용하시면 일괄적으로 적용이 가능합니다. ㅎㅎ